apps/finance/.solcover.js

```diff
@@ -1,6 +1,8 @@
 module.exports = {
-    norpc: true,
```

Comment:
I think we should keep the `norpc: true` flag, since we start our own rpc instance in the test script.